inference-engine/tests/functional/shared_test_classes/src/base/layer_test_utils.cpp

```diff
@@ -303,6 +305,17 @@ void LayerTestsCommon::ConfigureNetwork() {
             out.second->setPrecision(outPrc);
         }
     }
+
+    std::map<std::string, ngraph::PartialShape> shapes;
+    auto isdm = cnnNetwork.getInputsInfo();
+    for (auto&& idm : isdm) {
+        shapes[idm.first] = isdm[idm.first]->getInputData()->getPartialShape();
+        auto i = 0;
+        for (auto&& d : std::vector<ngraph::Dimension>(inputDynamicShape)) {
```

Comment:
Minor comment, replace to `std::vector<ngraph::Dimension>(inputDynamicShape).size()`?

Reply:
It can be reduced to a simple assignment. Thanks for your comment.